blog-content/using-github-as-a-headless-cms.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Using Github as a Headless CMS 
+createdAt: 2023-06-01T12:00:00Z
+readingTime: 12
+authorFirstName: Abhishek
+authorLastName: More
+authorTitle: Software Engineer
+authorTwitter: ''
+authorLinkedIn: 'https://www.linkedin.com/in/abhishek-more-linked/'
+authorGithub: 'https://github.com/Abhishek-More'
+authorWebsite: 'https://abhishekmore.com'
+authorPFP: 'https://tamuhack.org/static/th-2022/headshots/webp/abhishek.webp'
+tags: Highlight Engineering
+metaTitle: Using Github as a Headless CMS 
+---
+
+A content-management system (CMS) is software used to create, manage, and host content for blogs, docs, etc.
+It is crucial to use a CMS for any frequently updating content on your site, as it allows for:
+
+- Easy content creation and editing
+- Centralized content management
+- Collaboration and workflow management
+
+Recently, Highlight switched over from [Hygraph](https://hygraph.com) to GitHub for managing blog content, and we've been loving it!
```

Comment:
After this sentence, maybe just an intro sentence about what the blog will be about?